templates/suggest_social_connection.html

```diff
@@ -0,0 +1,144 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Add Social Account{% endblock %}
+
+{% block meta_description %}Add a social account to The Blue Alliance.{% endblock %}
+
+{% block content %}
+<div class="container">
+  {% if success %}
+  <div class="row">
+    <div class="col-xs-12 col-lg-6 col-lg-offset-3">
+      {% if success == '1' %}
+      <div class="alert alert-success">
+        <button type="button" class="close" data-dismiss="alert">&times;</button>
+        <h4>Thanks!</h4>
+        <p>We'll review your suggestion and get it added to the site soon!</p>
+      </div>
+      {% else %}
+      <div class="alert alert-danger">
+        <button type="button" class="close" data-dismiss="alert">&times;</button>
+        <h4>Uh oh!</h4>
+        <p>Sorry, but we are currently unable to support the URL you submitted. See below for currently supported formats.</p>
+      </div>
+      {% endif %}
+    </div>
+  </div>
+  {% endif %}
+  <div class="row">
+    <div class="col-xs-12 col-lg-6 col-lg-offset-3">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h1 class="panel-title">Add a team's social account to The Blue Alliance!</h1>
+        </div>
+        <div class="panel-body">
+          <p>Thanks for helping make The Blue Alliance better! Let us know about a social account so we can add them to the site!</p>
+          <ul>
+            <li>Your suggestion will be reviewed by a moderator</li>
+            <li>Your account info (like {{user_bundle.user.email}} and {{user_bundle.user.nickname}}) will also be submitted. <a href="{{user_bundle.logout_url}}">log in as a different user</a></li>
+          </ul>
+
+          <hr>
+
+          <p><strong>Currently supported formats are:</strong></p>
```

Comment:
If you wanted to get fancy, you could generate this from your `Type` const class.